docs/sources/tempo/configuration/_index.md

```diff
@@ -521,7 +524,7 @@ compactor:
         # Optional. Amount of data to buffer from input blocks. Default is 5 MiB.
         [v2_in_buffer_bytes: <int>]
 
-        # Optional. Flush data to backend when buffer is this large. Default is 30 MB.
+        # Optional. Flush data to backend when buffer is this large. Default is 20 MB.
```

Comment:
This fixes a documentation issue that was discussed here: https://github.com/grafana/helm-charts/pull/2171#discussion_r1093266710

which also addresses this issue: https://github.com/grafana/tempo/issues/2047